index.html

```diff
@@ -24,32 +32,24 @@
       let computerScore = parseInt(0);
 
       function playRound(playerSelection, computerSelection) {
-
-        if ((playerSelection === 'rock' && computerSelection === 'scissors') || (playerSelection === 'scissors' &&     computerSelection === 'paper') || (playerSelection === 'paper' && computerSelection === 'rock')) {
+        if ((playerSelection === 'rock' && computerSelection === 'scissors') || (playerSelection === 'scissors' && computerSelection === 'paper') || (playerSelection === 'paper' && computerSelection === 'rock')) { 
+          document.getElementById('roundResult').innerHTML = `You win this round, ${playerSelection} over computer ${computerSelection}. Try again`;
           playerScore++;
         } else if ((computerSelection === 'rock' && playerSelection === 'scissors') || (computerSelection === 'scissors' && playerSelection === 'paper') || (computerSelection === 'paper' && playerSelection === 'rock')) {
+          document.getElementById('roundResult').innerHTML = `You lost this round, ${computerSelection} over player ${playerSelection}. Try again`; 
           computerScore++;
         } else {
-          return;
+          document.getElementById('roundResult').innerHTML = `This round was a draw, player ${playerSelection} computer ${computerSelection}. Try again`;
         }  
       } 
-        
-      function game() {
-        for (let gameCount = 0; gameCount !== 6; gameCount++) { 
-          const playerSelection = prompt('Write either rock, paper, or scissors to play').toLowerCase();   
-          const computerSelection = computerPlay();
-          playRound(playerSelection, computerSelection);
-        }  
-      }
-      game();
-
-      if (playerScore > computerScore) {
-        console.log(`Computer ${computerScore} Player ${playerScore}. You WIN!!`);
-      } else if (playerScore < computerScore) {
-        console.log(`Computer ${computerScore} Player ${playerScore}. Computer wins!! Try again`);
-      } else {
-        console.log(`Computer ${computerScore} Player ${playerScore}. Its a draw`);
-      }
+
+      window.addEventListener('click', function(e) {
```

Comment:
I would put this onlick listener on to the play button, rather than the whole window